torch/csrc/jit/pybind_utils.h

```diff
@@ -430,6 +430,8 @@ inline IValue toIValue(
             }
             return repeated;
           }
+        case TypeKind::BoolType:
```

Comment:
Does this fix #26036? Add it to the PR description if so

Reply:
it is in the PR description :')

Reply:
looool, I had the exact same question! I'll wait till this in, this way I don't have to fix my PR.